script/yarn/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "yarn-dependency",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "devDependencies": {
+      "yarn": "^1.22.19"
+    }
+  }
+  
```

Comment:
Better add a new line at the end of file.

Reply:
or rather remove trainling white spaces at last line